Find a functionally equivalent code:android.view.ViewGroup.setClipChildren:COMMENT
Method Modifier: public      
Comment:/**
 * By default, children are clipped to their bounds before drawing. This
 * allows view groups to override this behavior for animations, etc.
 *
 * @param clipChildren true to clip children to their bounds,
 * false otherwise
 * @attr ref android.R.styleable#ViewGroup_clipChildren
 */

@@ -4,12 +4,13 @@
         setBooleanFlag(FLAG_CLIP_CHILDREN, clipChildren);
         for (int i = 0; i < mChildrenCount; ++i) {
             View child = getChildAt(i);
-            if (child.mDisplayList != null) {
-                child.mDisplayList.setClipToBounds(clipChildren);
+            if (child.mRenderNode != null) {
+                child.mRenderNode.setClipToBounds(clipChildren);
             }
         }
+        invalidate(true);
     }
 }

